src/decode.c

```diff
@@ -161,23 +167,35 @@ int decode_macs_table (msgpack_object *key, msgpack_object *val, schedule_t **t)
     uint32_t i;
     uint32_t count;
     msgpack_object *ptr = val->via.array.ptr;
-    mac_address *macs;
     (void ) key;
     
     count = val->via.array.size;
-    create_mac_table( *t, count );
-    macs = (*t)->macs;
+
+    if (count > MAXIMUM_BLOCKED_MAC_LIST) {
```

Comment:
We actually can have more then MAXIMUM_BLOCKED_MAC_LIST, just not in a single time window.  I don't think we want to validate this value here.

Reply:
I think the check should be against the `index` list size.  https://github.com/Comcast/aker/pull/79/files#diff-9981e0124d54e77417593d84a589ae47R234   val->via.array.size <= 128